modules/stepper/Stepper.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "@diana-ui/types";
+import { withStyles } from "@diana-ui/base";
+
+// @ts-ignore
+export interface IProps extends StandardProps<"div"> {
+  steps: number;
+  activeStep?: number;
+  clickable?: boolean;
+  onChange?: (stepIndex: number) => void;
+}
+
+const stylesheet: ThemeStyleSheetFactory = theme => ({
+  container: {
+    boxSizing: "border-box",
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between",
+    position: "relative"
+  },
+  stepWrapper: {
+    display: "flex",
+    flex: 1,
+    alignItems: "center",
+
+    "@selectors": {
+      "&:first-child": {
+        flex: 0
+      }
+    }
+  },
+  step: {
+    boxSizing: "border-box",
+    minWidth: 12,
+    minHeight: 12,
+    borderRadius: "50%",
+    backgroundColor: theme.colors.grey.grey50,
+    padding: 3,
+    display: "flex",
+    "@selectors": {
+      "&.active": {},
+      "&.clickable": {
+        cursor: "pointer"
+      }
+    }
+  },
+  innerStep: {
+    boxSizing: "border-box",
+    flex: 1,
+    borderRadius: "50%",
+    "@selectors": {
+      "&.active": {
+        backgroundColor: theme.colors.black
+      }
+    }
+  },
+  line: {
+    boxSizing: "border-box",
+    width: "100%",
+    height: 2,
+    backgroundColor: theme.colors.grey.grey50,
+    "@selectors": {
+      "&.active": {}
+    }
+  }
+});
+
+const Stepper: React.FC<IProps & WithStylesProps> = ({
+  cx,
+  styles,
+  className,
+  steps,
+  clickable = false,
+  onChange,
+  activeStep = 0
```

Comment:
if this prop is what makes `Stepper` parent controlled, I would not set the default here. A `initialActiveStep` is preferred IMO